.storybook/main.js

```diff
@@ -104,7 +104,17 @@ export default {
 				devSourcemap: configType === "DEVELOPMENT",
 			},
 			resolve: {
-				alias: components.map(component => ({ find: `@spectrum-css/${component}`, replacement: path.resolve(__dirname, `../components/${component}`) })),
+				alias: [
+					...components.map(component => ({ find: `@spectrum-css/${component}`, replacement: path.resolve(__dirname, `../components/${component}`) })),
```

Comment:
This gently expands our local alias'ing should we need it in future for more direct imports.